xarray/core/variable.py

```diff
@@ -660,7 +660,7 @@ def _broadcast_indexes_basic(self, key):
         dims = tuple(
             dim for k, dim in zip(key, self.dims) if not isinstance(k, integer_types)
         )
-        return dims, BasicIndexer(key), None
+        return dims, BasicIndexer(key, fastpath=True), None
```

Comment:
Can we remove this please? BasicIndexer is going away anyway...

Reply:
ok.